From e1298a9d5b23cfe07e2902f4da6f9e32fa5f3c2e Mon Sep 17 00:00:00 2001
From: I-Al-Istannen <i-al-istannen@users.noreply.github.com>
Date: Thu, 11 Dec 2025 20:14:32 +0100
Subject: [PATCH 8/8] Prefill chat.kitctf.de URL

---
 app/screens/server/index.tsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/screens/server/index.tsx b/app/screens/server/index.tsx
index 12f215aeb..5082cad62 100644
--- a/app/screens/server/index.tsx
+++ b/app/screens/server/index.tsx
@@ -91,7 +91,7 @@ const Server = ({
     const [displayName, setDisplayName] = useState<string>('');
     const [buttonDisabled, setButtonDisabled] = useState(true);
     const [preauthSecret, setPreauthSecret] = useState<string>('');
-    const [url, setUrl] = useState<string>('');
+    const [url, setUrl] = useState<string>('https://chat.kitctf.de');
     const [displayNameError, setDisplayNameError] = useState<string | undefined>();
     const [urlError, setUrlError] = useState<string | undefined>();
     const [preauthSecretError, setPreauthSecretError] = useState<string | undefined>();
-- 
2.52.0

