From 0a874f2728b1f41dc7f10a0724a7eb20320b81b8 Mon Sep 17 00:00:00 2001
From: I-Al-Istannen <i-al-istannen@users.noreply.github.com>
Date: Thu, 11 Dec 2025 20:14:32 +0100
Subject: [PATCH] Prefill chat.kitctf.de URL

---
 app/screens/server/index.tsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/screens/server/index.tsx b/app/screens/server/index.tsx
index 6b0515b00..de88e33d2 100644
--- a/app/screens/server/index.tsx
+++ b/app/screens/server/index.tsx
@@ -91,7 +91,7 @@ const Server = ({
     const [displayName, setDisplayName] = useState<string>('');
     const [buttonDisabled, setButtonDisabled] = useState(true);
     const [preauthSecret, setPreauthSecret] = useState<string>('');
-    const [url, setUrl] = useState<string>('');
+    const [url, setUrl] = useState<string>('https://chat.kitctf.de');
     const [displayNameError, setDisplayNameError] = useState<string | undefined>();
     const [urlError, setUrlError] = useState<string | undefined>();
     const [preauthSecretError, setPreauthSecretError] = useState<string | undefined>();
-- 
2.51.0

